pkg/engine/context/context.go

```diff
@@ -201,12 +218,39 @@ func (ctx *Context) AddImageDetails(kindInterface interface{}, specInterface int
 			img["name"] = image
 			img["tag"] = "latest"
 		}
-		imgs[imageName] = img
+		containerImgs[imageName] = img
```

Comment:
@JimBugwadia - Used the image name as the key to add the imgInfo object to the map.